whitelist.json

```diff
@@ -131,25 +131,6 @@
             "gauge.jenkins.node.vm.memory.total.used"
         ]
     },
-    {
-        "monitorType": "collectd/docker",
```

Comment:
We have to leave this block for `collectd/docker` because it doesn't use any of the common logic that you refactored (it runs in a Python collectd plugin).

The `docker-container-stats` below is good to remove though.